### ISSUE TITLE
Instagram API error Unsupported request - method type

### ISSUE THREAD

**Issue by mkrappitz** [in2code-de/instagram#41]
As of recently the API communication in the console commands seems to fail in version 8.0.0 of EXT:instagram in all our project we use this extension in on any TYPO3 version (10, 11, 12) and any php version (7.4, 8.2, 8.3). For example the error message for the instagram:refreshtoken console command:
"Die Ausführung von Task "Konsolenbefehle ausführen (scheduler)" ist fehlgeschlagen mit folgender Meldung: Client error: `GET https://graph.instagram.com/refresh_access_token?grant_type=ig_refresh_token&access_token=IGQW...DZD` resulted in a `400 Bad Request` response: {"error":{"message":"Unsupported request - method type: get","type":"IGApiException","code":100,"fbtrace_id":"Azq1XPBho9 (truncated...)"

This looks to me like an API change at Instagram.

The instagram:importfeed console command just fails with no specific error message:
"Die Ausführung von Task "Konsolenbefehle ausführen (scheduler)" ist fehlgeschlagen mit folgender Meldung: Task failed to execute successfully. Class: TYPO3\CMS\Scheduler\Task\ExecuteSchedulableCommandTask, UID: 3".

There was no change in the instagram account passwords or Facebook app.

**Comment by raphael-dan-ilea** [comment]
Related to this: https://github.com/in2code-de/instagram/issues/39 .

**Comment by dev3h** [comment]
I had the same issue on 2025/05/09. It shows me the error "Unsupported request—method type." I have no idea how to resolve it. Does anyone else have this issue?

**Comment by ricardorohrs** [comment]
I've been having the same problem for two weeks now. I get the error "Unsupported request — method type". I have no idea how to solve it. Any ideas?

**Comment by dev-tawhid** [comment]
i am also facing same issue? is there any solution ?

**Comment by aekiratli** [comment]
same ...

**Comment by abdelazizelbessa** [comment]
> same ...

same here jan 8 2026

**Comment by abdelazizelbessa** [comment]
i solve it you need to get permission from mate side go to the app the problem not in the code

**Comment by Mudassirraza912** [comment]
> i solve it you need to get permission from mate side go to the app the problem not in the code

@abdelazizelbessa what permission we need from meta? how you resolved this issue can you explain. i stuck here in same error. my personal account working but logging with some test account throwing  _'Unsupported request - method type: get',_ this error

**Comment by TKLucas** [comment]
You need to validate in the app's meta developer whether you've completed the access verification, which is usually located below the enterprise portfolio settings, in the basic settings. Only after they approve the access verification will they release the permissions and advanced access resources in the API.

**Comment by Tushar-Kapil** [comment]
I think we need to submit our business for meta tech provider verification only after that it starts working.

**Comment by Ignapag** [comment]
Same issue, the problem is that there is no fucking button to vrify meta tech provider, its a fucking pain in the ass

**Comment by Raghunadh9** [comment]
> Same issue, the problem is that there is no fucking button to vrify meta tech provider, its a fucking pain in the ass

@Ignapag @Tushar-Kapil  @TKLucas @Mudassirraza912   brooo, i found it.. go to https://developers.facebook.com/apps/{your_app_id}/settings/basic/?business_id={your_business_id}#verification-card

here, 

<img width="1401" height="444" alt="Image" src="https://github.com/user-attachments/assets/df626a5a-1754-460e-a525-88a6ce51009d" />


you need to click on view details and fill up the form that's it.. they are going to give you reply in 5 days

follow this for concise guidance: https://www.reddit.com/r/WhatsAppAPIDisabled/comments/1o8gksl/how_i_got_approved_as_a_meta_tech_provider_full/

**Comment by Ignapag** [comment]
@Raghunadh9 yes i saw it just after I post it lol, the problem was i was bugged and couldnt acces there, but i log in another window incognite and could thank god. I complain for nothing😂. Well not nothing, ive been trying to use IG and Facebook API since 2 months ago and still on sending verifications..... but well.
